TramsDataApi/Migrations/TramsDb/20220303130816_ProposedAcademyAdditionalFields.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace TramsDataApi.Migrations.TramsDb
+{
+    public partial class ProposedAcademyAdditionalFields : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            _ = migrationBuilder.Sql(@"CREATE  VIEW sdd.vw_proposed_academy_additional_fields
```

Comment:
Can you remove the comments from this view please, the table they are from implies the usage.